Makefile

```diff
@@ -55,6 +58,29 @@ CMAKE       ?= cmake
 # Misc
 Q           ?= @
 
+MACHINE_OS ?= $(shell uname -s)
+MACHINE_ARCH ?= $(shell uname -m)
+GOOS ?= linux
+GOARCH ?= amd64
```

Comment:
Can these please be indented to the same level as before and also replace all instances of `MACHINE_` with`UNAME_`.

Reply:
Done